src/Language/Haskell/Refact/Utils/GhcUtils.hs

```diff
@@ -81,6 +82,19 @@ everywhereM' f x
 
 -- ---------------------------------------------------------------------
 
+-- Monadic everywhereBut
+-- Traversal ceases if q holds for x
+
+everywhereButMStaged :: Monad m => SYB.Stage -> SYB.GenericQ Bool -> SYB.Generic m -> SYB.GenericM m
+everywhereButMStaged stage q f x
```

Comment:
Since GHC 7.10.1 we no longer need any of the `Staged` variants.  So if the original SYB version exists use that, or strip out the staging. I am slowly doing it as I touch the rest of the code too.